<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:test-suite start="1616394326431" stop="1616394407069" version="1.5.0" xmlns:ns2="urn:model.allure.qatools.yandex.ru">
    <name>Default suite : Default test</name>
    <title>Default suite : Default test</title>
    <test-cases>
        <test-case start="1616394331173" stop="1616394345328" status="passed">
            <name>Login</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="Default suite"/>
                <label name="testGroup" value="Default test"/>
                <label name="testClass" value="com.qait.tests.HomeworkTest"/>
                <label name="testMethod" value="Login"/>
                <label name="host" value="faizejafri-Vostro-15-3568"/>
                <label name="thread" value="139513@faizejafri-Vostro-15-3568.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1616394331173" stop="1616394345328" status="passed">
            <name>Login</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="Default suite"/>
                <label name="testGroup" value="Default test"/>
                <label name="testClass" value="com.qait.tests.HomeworkTest"/>
                <label name="testMethod" value="Login"/>
                <label name="host" value="faizejafri-Vostro-15-3568"/>
                <label name="thread" value="139513@faizejafri-Vostro-15-3568.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1616394345343" stop="1616394345866" status="passed">
            <name>Verify_Homework_Page</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="Default suite"/>
                <label name="testGroup" value="Default test"/>
                <label name="testClass" value="com.qait.tests.HomeworkTest"/>
                <label name="testMethod" value="Verify_Homework_Page"/>
                <label name="host" value="faizejafri-Vostro-15-3568"/>
                <label name="thread" value="139513@faizejafri-Vostro-15-3568.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1616394345343" stop="1616394345866" status="passed">
            <name>Verify_Homework_Page</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="Default suite"/>
                <label name="testGroup" value="Default test"/>
                <label name="testClass" value="com.qait.tests.HomeworkTest"/>
                <label name="testMethod" value="Verify_Homework_Page"/>
                <label name="host" value="faizejafri-Vostro-15-3568"/>
                <label name="thread" value="139513@faizejafri-Vostro-15-3568.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1616394345885" stop="1616394345970" status="passed">
            <name>verify_CreateHomework_Link_isDisplayed</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="Default suite"/>
                <label name="testGroup" value="Default test"/>
                <label name="testClass" value="com.qait.tests.HomeworkTest"/>
                <label name="testMethod" value="verify_CreateHomework_Link_isDisplayed"/>
                <label name="host" value="faizejafri-Vostro-15-3568"/>
                <label name="thread" value="139513@faizejafri-Vostro-15-3568.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1616394345885" stop="1616394345970" status="passed">
            <name>verify_CreateHomework_Link_isDisplayed</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="Default suite"/>
                <label name="testGroup" value="Default test"/>
                <label name="testClass" value="com.qait.tests.HomeworkTest"/>
                <label name="testMethod" value="verify_CreateHomework_Link_isDisplayed"/>
                <label name="host" value="faizejafri-Vostro-15-3568"/>
                <label name="thread" value="139513@faizejafri-Vostro-15-3568.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1616394346006" stop="1616394346810" status="passed">
            <name>verify_CreateHomework_link_redirects_Correctly</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="Default suite"/>
                <label name="testGroup" value="Default test"/>
                <label name="testClass" value="com.qait.tests.HomeworkTest"/>
                <label name="testMethod" value="verify_CreateHomework_link_redirects_Correctly"/>
                <label name="host" value="faizejafri-Vostro-15-3568"/>
                <label name="thread" value="139513@faizejafri-Vostro-15-3568.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1616394346006" stop="1616394346810" status="passed">
            <name>verify_CreateHomework_link_redirects_Correctly</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="Default suite"/>
                <label name="testGroup" value="Default test"/>
                <label name="testClass" value="com.qait.tests.HomeworkTest"/>
                <label name="testMethod" value="verify_CreateHomework_link_redirects_Correctly"/>
                <label name="host" value="faizejafri-Vostro-15-3568"/>
                <label name="thread" value="139513@faizejafri-Vostro-15-3568.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1616394346821" stop="1616394346999" status="failed">
            <name>verify_Next_Button</name>
            <failure>
                <message>AssertionError: expected [true] but found [false]</message>
                <stack-trace>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.qait.tests.HomeworkTest.verify_Next_Button(HomeworkTest.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="Default suite"/>
                <label name="testGroup" value="Default test"/>
                <label name="testClass" value="com.qait.tests.HomeworkTest"/>
                <label name="testMethod" value="verify_Next_Button"/>
                <label name="host" value="faizejafri-Vostro-15-3568"/>
                <label name="thread" value="139513@faizejafri-Vostro-15-3568.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1616394346821" stop="1616394346999" status="failed">
            <name>verify_Next_Button</name>
            <failure>
                <message>AssertionError: expected [true] but found [false]</message>
                <stack-trace>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.qait.tests.HomeworkTest.verify_Next_Button(HomeworkTest.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="Default suite"/>
                <label name="testGroup" value="Default test"/>
                <label name="testClass" value="com.qait.tests.HomeworkTest"/>
                <label name="testMethod" value="verify_Next_Button"/>
                <label name="host" value="faizejafri-Vostro-15-3568"/>
                <label name="thread" value="139513@faizejafri-Vostro-15-3568.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1616394347007" stop="1616394407037" status="failed">
            <name>Verify_Fields_CreateHomework</name>
            <failure>
                <message>AssertionError: [ASSERT FAILED]: Element fields_Selectable not found on the Homework !!!</message>
                <stack-trace>java.lang.AssertionError: [ASSERT FAILED]: Element fields_Selectable not found on the Homework !!!
	at org.testng.Assert.fail(Assert.java:94)
	at com.qait.automation.getpageobjects.GetPage.element(GetPage.java:146)
	at com.qait.keywords.HomeworkAction.verify_SelectionFields(HomeworkAction.java:60)
	at com.qait.tests.HomeworkTest.Verify_Fields_CreateHomework(HomeworkTest.java:44)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="Default suite"/>
                <label name="testGroup" value="Default test"/>
                <label name="testClass" value="com.qait.tests.HomeworkTest"/>
                <label name="testMethod" value="Verify_Fields_CreateHomework"/>
                <label name="host" value="faizejafri-Vostro-15-3568"/>
                <label name="thread" value="139513@faizejafri-Vostro-15-3568.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1616394347007" stop="1616394407037" status="failed">
            <name>Verify_Fields_CreateHomework</name>
            <failure>
                <message>AssertionError: [ASSERT FAILED]: Element fields_Selectable not found on the Homework !!!</message>
                <stack-trace>java.lang.AssertionError: [ASSERT FAILED]: Element fields_Selectable not found on the Homework !!!
	at org.testng.Assert.fail(Assert.java:94)
	at com.qait.automation.getpageobjects.GetPage.element(GetPage.java:146)
	at com.qait.keywords.HomeworkAction.verify_SelectionFields(HomeworkAction.java:60)
	at com.qait.tests.HomeworkTest.Verify_Fields_CreateHomework(HomeworkTest.java:44)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="Default suite"/>
                <label name="testGroup" value="Default test"/>
                <label name="testClass" value="com.qait.tests.HomeworkTest"/>
                <label name="testMethod" value="Verify_Fields_CreateHomework"/>
                <label name="host" value="faizejafri-Vostro-15-3568"/>
                <label name="thread" value="139513@faizejafri-Vostro-15-3568.main(1)"/>
            </labels>
        </test-case>
    </test-cases>
    <labels>
        <label name="framework" value="TestNG"/>
        <label name="language" value="JAVA"/>
    </labels>
</ns2:test-suite>
